### PR TITLE
 CRUD for Job Descriptions complete with restful routes. fixes #8

### LINKED ISSUE #8: Job Description:API for crud  (Abha Rana)

### PR DESCRIPTION
**Pull request by abharana24**
fixes #8

### CODE REVIEW

**Comment by sonalichawla16** [comment]
Please resolve conflicts

File: api/routes/index.js [hunk 1]
```diff
@@ -3,8 +3,17 @@ module.exports=(app) =>
 { 
     // Sample get route
     // app.get('/login', controller.);
+
+    //Routes for Job Description
+    app.post('/api/jobDescription',controller.jobDescription.createJd);
+  	app.get('/api/jobDescription',controller.jobDescription.showAllJds);
+	  app.get('/api/jobDescription/:id',controller.jobDescription.showJd);
```
Comment (vishivish18) [inline api/routes/index.js:10]: Why do we have this indentation here ?

Comment (abharana24) [reply api/routes/index.js:10]: this is showing properly indented in VS Code. will recheck this and resolve other conflicts and push the code again.

File: api/routes/index.js [hunk 2]
```diff
@@ -9,6 +9,15 @@ module.exports=(app) =>
     app.delete('/api/employee/:id', controller.employee.remove);
     // Sample get route
     // app.get('/login', controller.);
+
+    //Routes for Job Description
+    app.post('/api/jobDescription',controller.jobDescription.createJd);
+	app.get('/api/jobDescription',controller.jobDescription.showAllJds);
+	app.get('/api/jobDescription/:id',controller.jobDescription.showJd);
```
Comment (sonalichawla16) [inline api/routes/index.js:16]: It seems to be indentation issue

**Pull request merged**